dotnet/src/Connectors/Connectors.OpenAI/Files/OpenAIFileService.cs

```diff
@@ -109,11 +109,22 @@ public async Task DeleteFileAsync(string id, CancellationToken cancellationToken
     /// <remarks>
     /// Files uploaded with <see cref="OpenAIFilePurpose.Assistants"/> do not support content retrieval.
     /// </remarks>
-    public BinaryContent GetFileContent(string id, CancellationToken cancellationToken = default)
+    public async Task<BinaryContent> GetFileContentAsync(string id, CancellationToken cancellationToken = default)
     {
         Verify.NotNull(id, nameof(id));
+        var (stream, mimetype) = await this.StreamGetRequestAsync($"{this._serviceUri}/{id}/content", cancellationToken).ConfigureAwait(false);
 
-        return new BinaryContent(() => this.StreamGetRequestAsync($"{this._serviceUri}/{id}/content", cancellationToken));
+        using (stream)
+        {
```

Comment:
![image](https://github.com/microsoft/semantic-kernel/assets/66376200/97f55cbd-526e-4d89-97cf-9c1570fa4a38)

I'd assert this would be every scenario (why copy stream if not required).  In any case, is there a tracking issue for this follow-up?

Reply:
> I'd assert this would be every scenario (why copy stream if not required). In any case, is there a tracking issue for this follow-up?

Not sure I understand your message (why copy stream if not required).